src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformArtifactoryFormHelper.ts

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+import * as Yup from 'yup'
+import { cloneDeep, unset, size, isUndefined } from 'lodash-es'
+import { TerraformStoreTypes } from '../TerraformInterfaces'
+
+export const formatInitialValues = (isConfig: boolean, prevStepData: any, isTerraformPlan: boolean) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232248)
```suggestion
export const formatInitialValues = (isConfig: boolean, prevStepData: unknown, isTerraformPlan: boolean) => {
```